src/main/java/io/nats/client/Nats.java

```diff
@@ -393,6 +397,19 @@ private Nats() {}
     static final String TCP_SCHEME = "tcp";
     static final String TLS_SCHEME = "tls";
 
+    protected static final String SYSTEM_ENV_MSG_DELIVERY_THREAD_POOL_SIZE = "JNATS_MSG_DELIVERY_THREAD_POOL_SIZE";
+    static private MsgDeliveryPool globalMsgDeliveryPool = null;
+
+    static {
+        final String msgDeliveryPoolSizeStr = System.getenv(SYSTEM_ENV_MSG_DELIVERY_THREAD_POOL_SIZE);
+        if (msgDeliveryPoolSizeStr != null) {
+            int poolSize = 0;
+            try { poolSize = Integer.parseInt(msgDeliveryPoolSizeStr); } catch (Exception e) {}
```

Comment:
Is this just to catch `blank`? should we do something with the exception if it's a non-blank non-int?

Reply:
I am planning to remove creation of the pool from system property and change `Nats.MsgDeliveryThreadPoolSize()` with `Nats.createMsgDeliveryThreadPool()` to remove any idea that pool could be resized. Will update the PR in a bit.